num_to_words/__init__.py

```diff
@@ -29,9 +31,7 @@ def num_to_word(num, lang, separator=", ", combiner=" "):
     for language in SUPPORTED_LANGUAGES:
         for num_index in range(10):
             num = num.replace(DIGITS_LANG_MAPPING[language][num_index], DIGITS_LANG_MAPPING["en"][num_index])
-
-    # Assert that input contains only integer number
-    assert num, "Input string is empty"
```

Comment:
why the assertation has been removed?

Reply:
Above I added
```python
    if not num:
        return ''
```
to return empty string if provided.